packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1260,4 +1260,18 @@
       <protected>false</protected>
       <shortDescription>Something went wrong and we couldn't generate an answer.</shortDescription>
     </labels>
+    <labels>
+      <fullName>quantic_ShowingResultsFor</fullName>
+      <value>Showing results for {{0}}</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Showing results for example</shortDescription>
+  </labels>
+  <labels>
+    <fullName>quantic_SearchInsteadFor</fullName>
+    <value>Search instead for</value>
+    <language>en_US</language>
+    <protected>false</protected>
+    <shortDescription>Search instead for</shortDescription>
+  </labels>
```

Comment:
Hello @cmartel-coveo ! These are the labels we display in Quantic when the query is updated by a pipeline trigger:

<img width="1212" alt="labels" src="https://github.com/coveo/ui-kit/assets/86681870/46311476-fdcc-4eb7-805b-21f542100508">

Please let me know if you think anything we can improve with these labels! Thank you!

Reply:
@mmitiche Sorry for the late reply... But your copy is good, so no worries :)